### ISSUE TITLE
nextcloud totp support for passman

### ISSUE THREAD

**Issue by versd** [nextcloud/passman-webextension#235]
### Steps to reproduce
1.  Enable TOTP in nextcloud user account
2.  Login to nextcloud server in passman settings
3.  Receive 'unauthorized [401]'  error

### Expected behaviour
TOTP code should be requested along with master password

### Actual behaviour
No TOTP code is requested and thus login to server fails

### Configuration
**Operating system**:  Windows 8.1

**Browser**: Chrome

**Extensions that might cause interference**: 

**Passman version**:  2.1.4

**Extension version**: 2.0.4

**Nextcloud version**: 13

**Comment by iudex-** [comment]
please have a look at
https://github.com/nextcloud/passman-webextension/issues/180
and
https://github.com/nextcloud/passman-webextension/wiki/Passman-extension-with-OTP

**Comment by versd** [comment]
I'm sorry, rtfm I guess :) thank you for the reply!

On Feb 16, 2018 7:00 PM, Nils <notifications@github.com> wrote:

please have a look at
#180<https://github.com/nextcloud/passman-webextension/issues/180>
and
https://github.com/nextcloud/passman-webextension/wiki/Passman-extension-with-OTP

**Comment by MhhhxX** [comment]
This doesn't seem to work for me in firefox. I created an app password and used it in the setup for the webextension.
After unlocking with the master password no credentials show up. The badge always shows zero.
I debugged the extension and it shows a 303 response when the extension tries to call the api (/index.php/apps/passman/api/v2/vaults/xxxxxxxxxxxxx). The response contains the html page asking for the totp.
